### PR TITLE
Remove italics for Farsi language blocks

### LINKED ISSUE #1731: No italics for Arabic-script language blocks
**Is your feature request related to a problem? Please describe.**
As a reader of an Arabic-script site (Persian, Arabic, Urdu, Pashto)
I never want to see italics
So that I can read the text

Given I am on an article that uses an Arabic script
And one of the fragment blocks contains the italic attribute
Then I see that fragment as if it did not have an italic attribute

Given I am on an Arabic-script article
Then I see no italic text

Note:
- Exclusion for nested blocks: e.g. `fa` article with  `en-gb` block - italic should work in the `en-gb` block but not for the `fa` blocks. 

**Describe the solution you'd like**

- [ ] Pull in [work in psammead which disabled italics in persian](https://github.com/bbc/psammead/blob/latest/packages/utilities/psammead-styles/src/__snapshots__/font-styles.test.js.snap#L59-L65)
* this means:
  * Updating `psammead-styles` to version `1.1.0` if/where needed ([see work in psammead to de-italicise caption](https://github.com/bbc/psammead/pull/939))
  * [using functions like `getSansItalic` where we currently use `font-style: italic`](https://github.com/bbc/psammead/blob/799c4c667098094e1acd801ee2b51f4edf5433bb/packages/components/psammead-caption/src/index.jsx#L22)
 * Rethinking our approach to `<i>` and `<b>` tags in [`Fragment`](https://github.com/bbc/simorgh/blob/2b72dbef65d90018dcdde9f04e3058a404510422/src/app/containers/Fragment/index.jsx)

**Describe alternatives you've considered**
Fixing these issues by using
```
 [lang=fa] & {
    font-style: normal;
  }
```
but the approach in this ticket removes duplication and will handle this for more languages than just farsi.

**Testing notes**
-  [Farsi article c19v65z3lg5o](http://localhost.bbc.co.uk:7080/persian/articles/c19v65z3lg5o) has italics in the headline in the data (on  `فوتبال` )

Regression is sufficient & manual testing. 

Dev insight: Unit tests should be sufficient.

**Additional context**

### PR DESCRIPTION
**Pull request by twinlensreflex**
Resolves #1731

**Overall change:**

- [x] no italics on farsi article
- [ ] english blocks on farsi language page unaffected
- [ ] tests added

**Code changes:**

- if article language is Farsi blocks are filtered and italic attributes removed

**Testing notes:**

-  [Farsi article c19v65z3lg5o](http://localhost.bbc.co.uk:7080/persian/articles/c19v65z3lg5o) has italics in the headline in the data (on  `فوتبال` ) and this word will show up in italics without these code changes

**Before**
soccer word in farsi italic
**After**
soccer word in farsi - not italic

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Comment by twinlensreflex** [comment]
https://github.com/bbc/psammead/issues/376 linked

**Review by oluoluoxenfree — APPROVED** [review]

File: src/app/lib/globalStyles/index.jsx [hunk 1]
```diff
@@ -19,6 +19,12 @@ const GlobalStyleComponent = createGlobalStyle`
       box-sizing: inherit;
     }
 
+    i {
```
Comment (dr3) [inline src/app/lib/globalStyles/index.jsx:22]: Do we need this? the new font styles work, thats yet to be pulled into simorgh, ensures that persian cant be italic I think https://github.com/bbc/psammead/blob/latest/packages/utilities/psammead-styles/src/__snapshots__/font-styles.test.js.snap#L59-L65

Comment (dr3) [reply src/app/lib/globalStyles/index.jsx:22]: @sadick254 is above right?

**Review by dr3 — CHANGES_REQUESTED** [review]

File: src/app/lib/globalStyles/index.jsx (hunk 1 above)
Comment (ghost) [reply src/app/lib/globalStyles/index.jsx:22]: @dr3 Yes thats right, `persian` doesn't have `boldItalic` defined and it should fall back to just `bold`

**Comment by ghost** [comment]
Think it would be better to update https://github.com/bbc/simorgh/blob/33d5b8110a0d6977fb4a957975c19d67ceacfc7b/src/app/containers/Fragment/index.jsx#L3-L6
to use the font functions in `psammead-styles/font-styles` instead of `[lang=fa]`

**Comment by twinlensreflex** [comment]
closed on https://github.com/bbc/simorgh/issues/2618

**Pull request closed without merging**